src/components/screens/vacation-requests-screen.tsx

```diff
@@ -210,22 +268,19 @@ const VacationRequestsScreen = () => {
     try {
       setLoading(true);
       const vacationRequestId = selectedRowId as string;
-      const createdVacationRequestStatus =
-        await vacationRequestStatusApi.createVacationRequestStatus({
-          id: vacationRequestId,
-          vacationRequestStatus: {
-            vacationRequestId: vacationRequestId,
-            status: newStatus,
-            message: LocalizationUtils.getLocalizedVacationRequestStatus(newStatus),
-            createdAt: new Date(),
-            createdBy: userProfile.id,
-            updatedAt: new Date(),
-            updatedBy: userProfile.id
-          }
-        });
-
+      const createdVacationRequestStatus = await vacationRequestStatusApi.createVacationRequestStatus({
```

Comment:
In this case have a new line after the = so the await is on the same line as the request. IMO doing it this was is clearer

Reply:
Added new line to make it clearer.